### PR TITLE
finish todo list

### PR DESCRIPTION
**Pull request by danielufeli**
### What does this PR do?
- Add a fuctionality for status update
- Add javascript functions to mark task as complete
- Add Linters

### CODE REVIEW

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
.

File: src/index.js [hunk 1]
```diff
@@ -73,8 +73,8 @@ const component = () => {
     } else {
       todos = JSON.parse(localStorage.getItem('todos'));
     }
-    const m = todos.filter((x) => x.index.toString() !== todo.id);
-    const reset = m.map((e, i) => {
+    const filterArr = todos.filter((x) => x.index.toString() !== todo.id);
+    const reset = filterArr.map((e, i) => {
```
Comment (MrBrN197) [inline src/index.js]: - [x] Your "clear completed" functionality is working correctly although the functionality for this should be in a different module file that you then import into this file.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @danielufeli  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
Hi @danielufeli 

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**